docs/designers-developers/designers/animation.md

```diff
@@ -28,6 +28,13 @@ In creating consistent animations, we have to establish physical rules for how e
 
 Reuse animations if one already exists for your task. 
 
+## Accessibility Considerations
+
+- Animations should be subtle. Be cognizent of users with [motion triggering vestibular disorders](https://www.ncbi.nlm.nih.gov/pubmed/29017000).
```

Comment:
This feels like awkward phrasing, what about
> Animations should be subtle. Be cognizant of users with [vestibular disorders triggered by motion.](https://www.ncbi.nlm.nih.gov/pubmed/29017000)

Reply:
That's much better, thanks! 👍